### PR TITLE
[feature](fe) add function 'BitmapAgg' in nereids

### PR DESCRIPTION
**Pull request by mrhhsg**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mrhhsg** [comment]
run buildall

**Comment by mrhhsg** [comment]
run buildall

**Review by yiguolei — DISMISSED** [review]
LGTM

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/BitmapAgg.java
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/agg/BitmapAgg.java:1]: please add regression test case for all signatures in regression-test/suites/nereids_function_p0/agg_function/agg.groovy

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**